skt/executable.py

```diff
@@ -540,6 +540,15 @@ def setup_parser():
         type=str,
         help="Base repo URL"
     )
+    parser_merge.add_argument(
+        "--extra-git-args",
+        type=int,
+        help=(
+            "extra arguments to pass to `git fetch` "
+            "(ex: --depth 1 for shallow clone)")
+        ,
```

Comment:
The last ending parenthesis should be on this line, before the `,`(pycodestyle)

Reply:
Ah, good catch. I'll fix that.